en/docs/observe/mi-observe/working-with-management-api.md

```diff
@@ -1310,7 +1713,33 @@ The management API has multiple resources to provide information regarding the d
 	}
 
 	```	
-	
+
+	??? note "Search for a log configuration by name"
```

Comment:
let's make it 'log files' instead of 'log configuration' because this resource is to search for a specific log file